src/tasks/block.rs

```diff
@@ -177,23 +181,25 @@ impl BlockBuilder {
                 loop {
                     // sleep the buffer time
                     tokio::time::sleep(Duration::from_secs(self.secs_to_next_target())).await;
-                    tracing::trace!("beginning block build cycle");
+                    tracing::info!("beginning block build cycle");
 
                     // Build a block
                     let mut in_progress = InProgressBlock::default();
                     self.get_transactions(&mut in_progress).await;
-                    self.get_bundles(&mut in_progress).await;
+
+                    // TODO: Implement bundle ingestion #later
```

Comment:
let's add a linear ticket for this

Reply:
this work should be tracked somewhere i think? cc @dylanlott